app/controllers/api/v1/set_member_subjects_controller.rb

```diff
@@ -12,8 +12,11 @@ def create
     super do |sms|
       set_id = sms.subject_set_id
       update_set_counts(set_id)
-      linked_workflow_ids(set_id).each do |workflow_id|
-        SubjectWorkflowStatusCreateWorker.perform_async(
+      set_ids = linked_workflow_ids(set_id)
```

Comment:
this is the workflow ids - so doesn't give you an idea about how many SMS records we are creating. These are going to be pretty small, normally 1 so i think these can either be left as is or converted to use something like, make it between now and 1 minute or something?

Reply:
oops, got carried away there. Made the duration somewhere within a minute on this one.